test/modules/CloudToDeviceMessageTester/CloudToDeviceMessageReceiver.cs

```diff
@@ -86,10 +88,28 @@ public async Task StartAsync(CancellationToken ct)
                     try
                     {
                         Message message = await this.deviceClient.ReceiveAsync();
+
+                        if (message == null)
+                        {
+                            this.logger.LogWarning("Received message is null");
+                            continue;
+                        }
+
+                        if (!(message?.Properties.ContainsKey(TestConstants.Message.SequenceNumberPropertyName) ?? false) ||
```

Comment:
Since the message cannot be null here, I don't think we need the `?.`

Reply:
It is possible to be null as operation timeout happens or edgeHub restart.